package.json

```diff
@@ -237,7 +237,7 @@
       "The 'versionToTagMappings' section defines the mapping between released tags for our CLI plugin and the org version that each tag supports."
     ],
     "target": {
-      "versionNumber": "65.0",
+      "versionNumber": "66.0",
       "matchingDevServerVersion": "~13.0.27"
```

Comment:
Can you please bump the dependency version and matching devServerVersion in a follow up PR?